packages/react-form/src/hooks/list/list.ts

```diff
@@ -216,6 +216,9 @@ export function useList<Item extends object>(
                 }
                 validate();
               },
+              setError(error: string) {
+                dispatch(updateErrorAction({target, error}));
+              },
```

Comment:
Why this change?

Reply:
The test for submitting list failed because the imperative method for setting an error on list fields was missing (probably didn't pull it out of my codesandbox version).

Reply:
This implementation was missing but it was only caught when I un-skipped the form tests. I can open a separate PR for it, but thought I'd just squeeze it in here alongside its tests.